controllers/cluster.weave.works/existinginframachine_controller.go

```diff
@@ -470,18 +433,30 @@ func (a *ExistingInfraMachineReconciler) update(ctx context.Context, c *existing
 	node, err := a.findNodeByPrivateAddress(ctx, addr)
 	if err != nil {
 		if apierrs.IsNotFound(err) {
+			log.Infof("no existing node found for %s...", addr)
+			var nodes corev1.NodeList
+			if err := a.Client.List(ctx, &nodes); err == nil {
+				nodeIPs := []string{}
+				for _, node := range nodes.Items {
+					nodeIPs = append(nodeIPs, getNodePrivateAddress(&node))
+				}
+				log.Infof("all existing nodes found: %+v", nodeIPs)
+			}
```

Comment:
This block appears to only be used for logging the existing nodes.  Nit - if that is the case, move to a separate function to make the code easier to read.

Reply:
K